src/views/addManager/addManager.js

```diff
@@ -0,0 +1,280 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Form, Field, Formik } from "formik";
+import * as Yup from "yup";
+import * as axios from "axios";
+import { AddProperty } from '../addProperty/addProperty';
+import { SearchPanel, SearchPanelVariant } from "react-search-panel";
+import UserContext from "../../UserContext";
+import Button from "../../components/Button";
+import Modal from '../../components/Modal';
+import Toast from '../../utils/toast';
+import useMountEffect from '../../utils/useMountEffect';
+import './addManager.scss';
+import RoleEnum from '../../Enums/RoleEnum';
+
+const validationSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a First Name"),
+  lastName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a Last Name"),
+  phone: Yup.string()
+    .min(
+      5,
+      "*Number must contain at least 5 digits to be a valid phone/text number"
+    )
+    .max(20, "*Numbers can't be longer than 20 digits")
+    .required("*a valid phone number is required"),
```

Comment:
Is there a reason the validation does length 5 - 20? Accounting for non-standard/international #?

Reply:
The validation length was copied from another component in the app. It looks like there is some inconsistency across different components. Phone minimum lengths are 5 or 7 and the maximum lengths are 20 or 25. Different components use different combinations. 

Maybe it is a good idea to open an issue on this - something like having a common file in which to declare properties for use in the call to `Yup.object().shape()`?

Reply:
@ccbass @DaveTrost 
There is also an inconsistency between the frontend and backend on this, so maybe this is something we should bring up and resolve in the Wednesday meeting.

I think Nick made a pretty compelling argument for increasing the maximum length to 30 in [backend PR 183](https://github.com/codeforpdx/dwellinglybackend/pull/183). 

Given that any non-numeric characters are still counted toward the max and min lengths, and given the talk last meeting about this software potentially being used by other organizations down the line, I think it's probably better to future-proof it now by having a buffer between a standard number's characters and the field limits.